Formula/butterfly-cli-package.rb

```diff
@@ -0,0 +1,24 @@
+class ButterflyCliPackage < Formula
+  desc "Application code transformation tool"
+  homepage "https://github.com/paypal/butterfly"
+  url "https://search.maven.org/remotecontent?filepath=com/paypal/butterfly/butterfly-cli-package/2.2.0/butterfly-cli-package-2.2.0.zip"
```

Comment:
Is there anyway to make the version URI element and the version zip file name suffix dynamic? The idea is to avoid the need to change this line every time a new version is released.

Reply:
Not sure that would make sense. A package manager kind of has to know the _version_ to be stable.